### PR TITLE
COMMERCE-10806 headless-commerce-delivery-order-test 

### PR DESCRIPTION
**Pull request by ghost**
https://issues.liferay.com/browse/COMMERCE-10806

resent from:
https://github.com/brianchandotcom/liferay-portal/pull/133453#issuecomment-1511384129

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-order-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/order/resource/v1_0/test/PlacedOrderItemResourceTest.java [hunk 1]
```diff
@@ -113,6 +134,16 @@ public void setUp() throws Exception {
 				1, 2022, 12, 0, 0, 0, 0, 0, 0, true, _serviceContext);
 	}
 
+	@Override
+	@Test
+	public void testGetPlacedOrderItem() throws Exception {
+		super.testGetPlacedOrderItem();
+
+		_testGetPlacedOrderItemWithFileEntry();
+
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-order-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/order/resource/v1_0/test/PlacedOrderItemResourceTest.java:143]: not sure about this blank line, I hope Brian don't mind if it is not right :)!

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM!

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**